docs/configuring_prometheus.md

```diff
@@ -0,0 +1,36 @@
+# Configuring Prometheus for better Performance with Promscale
+
+This document mentions details about configuring Prometheus's remote-write settings to maximize performance
+from Promscale.
+
+Prometheus is a monitoring tool, that scrapes targets and sends scraped metric data to remote-storage systems
+like Promscale. The samples are pushed by Prometheus through the remote-write component of Prometheus. You
+can configure the remote-write component based on the requirements and tune the component accordingly. Details
+related to the remote-write configuration can be found [here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
+
+Remote-write performance is mainly dependent on `queue_config`. Changing values in `queue_config` will allow
+the component to adjust to different scenarios but ideally they should be:
+
+### capacity
+
+This corresponds to the maximum number of samples that the queues in remote-write shards can hold. If you have
+a high-throughput environment, capacity of queues should be `10000` at least.
+
+### minimum number of shards
+
+Shards are they actual parts of remote-write component that push data to remote-write endpoint (concurrently). More the number of
+shards, more would be the parallel requests to remote-write. Since Promscale is optimized for concurrent inserts,
+it will perform better with more shards. Ideally, you can set `min_shards: 4`. However, you should also keep in mind that increasing shards by
+large values can impact the memory usage significantly.
+
+### batch send deadline
+
+Batch send deadline denotes the maximum time allowed for a particular write request to remote-write endpoint. You should
+set this high if you have a higher cardinality. Ideally, it should be set to `batch_send_deadline: 30s`. Also, do remember
+to increase the `remote_timeout` value in case `batch_send_deadline` exceeds it.
+
+### maximum retry delay
+
+The remote-write component implements backoff duration on requests to write-endpoint i.e., retry the write request again
+after a pause. This can be particularly useful if the remote-storage is facing some rate-limiting. So, you can set this
+to `1s`, however you can set higher values, depending on the requirements.
```

Comment:
Add the field key here as well as mentioned in above sections for other configurations. i.e. `max_backoff`

Reply:
I agree with this.